presidio-anonymizer/presidio_anonymizer/entities/analyzer_results.py

```diff
@@ -54,11 +54,12 @@ def _remove_conflicts(self):
         :return: List
         """
         unique_elements = []
+        other_elements = AnalyzerResults(self)
         for result_a in self:
-            other_elements = AnalyzerResults(self)
             other_elements.remove(result_a)
             if not any([result_a.has_conflict(other_element) for other_element in
                         other_elements]):
+                other_elements.append(result_a)
```

Comment:
why return an already checked element back?

Reply:
Because others need to compare themselves with it..
we checked one side that result_a is not contained in another result
But we need to make sure other results are not contained in result_a.